app/models/option_node.rb

```diff
@@ -296,6 +297,12 @@ def shortcode
     @shortcode = Base36.to_padded_base36(sequence, length: shortcode_length)
   end
 
+  def max_sequence
+    # for some reason ancestry scopes requests through the current node even if you don't
+    # call where through self, so you need to explicitly call unscoped here
```

Comment:
It wasn't so much a catch as a long, headache-inducing, painful debugging session -- the comment's there because calling `unscoped` is really weird and feels hacky and I wanted to make absolutely 100% sure it was clear why that was happening to save somebody else the trouble of removing it and then wondering why sequence assignment was broken.